manifests/reporting/querying/configmap.yaml

```diff
@@ -7,8 +7,22 @@ data:
     {
       "list": [
         {
-          "name": "resources",
+          "name": "resources-limit",
```

Comment:
Please move this rules in `helm/reporting-chart/templates/querying/configmap.yaml`